code/src/java/pcgen/system/ConsoleUIDelegate.java

```diff
@@ -109,37 +108,25 @@ public void showWarningMessage(String title, String message)
 		Logging.log(Logging.WARNING, title + " - " + message);
 	}
 
-	/**
-	 * {@inheritDoc}
-	 */
 	@Override
 	public boolean showGeneralChooser(ChooserFacade chooserFacade)
 	{
 		return false;
 	}
 
-	/**
-	 * {@inheritDoc}
-	 */
 	@Override
 	public String showInputDialog(String title, String message, String initialValue)
 	{
 		return null;
 	}
 
-	/**
-	 * {@inheritDoc}
-	 */
 	@Override
 	public CustomEquipResult showCustomEquipDialog(CharacterFacade character, 
 		EquipmentBuilderFacade equipBuilder)
 	{
 		return CustomEquipResult.CANCELLED;
 	}
 
-	/**
-	 * {@inheritDoc}
```

Comment:
The checkstyle setup should change: similar docs are automatically produced and it adds to the overhead of understanding what a given function is doing (every comment adds a small amount of overhead to parse and maybe ignore).